ios/FluentUI/SegmentedControl/SegmentedControl.swift

```diff
@@ -130,8 +133,13 @@ open class SegmentedControl: UIControl {
             }
         }
 
-        var segmentTextStyle: TextStyle {
-            return .subhead
+        var segmentTextFont: UIFont {
+            switch self {
+            case .tabs:
+                return TextStyle.subhead.font
+            case .primaryPill, .onBrandPill:
+                return UIFont.systemFont(ofSize: 16, weight: .regular)
```

Comment:
probably my fault, but we shouldn't need to specify .regular just deal with size param